### ISSUE TITLE
how to perform loss for Generating face from sketches 

### ISSUE THREAD

**Issue by XD666** [eladrich/pixel2style2pixel#16]
I assume the input is the sketch image (A) but the generated image (B) is a normal face. Is the LPIPS loss performed between A and B? It seems doesn't make sense for me. Could you please clarify more here regarding the loss perform?

**Comment by yuval-alaluf** [comment]
Hi @XD666 ,
The sketch generation task is a supervised task with pairs `(Input_sketch , Gt_image)`. Here, pSp receives as input `Input_sketch` and outputs a photo image `Output_image`.  Then, the losses are computed between `Gt_image` and `Output_image`.

**Comment by XD666** [comment]
Thank you for your clarification! It's a supervised task with paired images. That makes a lot of sense. Amazing work!

**Comment by simplecode123** [comment]
I want to train psp model on pair image use uses ---stylegan_weights, like in issue #39, but when I use this script 
```
projector.py --outdir=out --target=~/mytargetimg.png \
    --network=https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/ffhq.pkl 
```
from [stylegan2-ada-pytorch](https://github.com/dvschultz/stylegan2-ada-pytorch) it is work too slow :disappointed:
